.github/ISSUE_TEMPLATE/mod_integration.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Mod integration 
+about: Suggest an integration for a mod
+labels: enhancement, "mod-integration"
+assignees: 'Nerdpie'
```

Comment:
This line is a bit tongue-in-cheek, since I feel like I'm suggesting a lot of integrations, and then writing them myself.

Reply:
I'm _generally_ inclined to leave off assignments unless it's something only one person is capable of dealing with (say, highly technical changes).

Reply:
That's fine, I'll remove the 'assignees' line and update the labels.